pkg/util/queryutil/query.go

```diff
@@ -0,0 +1,10 @@
+package queryutil
```

Comment:
This package doesn't match the naming convention we have in `pkg/util`. Maybe just move it to the `util` package, since we have `pkg/util/hash_fp.go` already.